shared/src/libs/packages/http/libs/enums/http-code.enum.ts

```diff
@@ -2,6 +2,8 @@ const HTTPCode = {
   OK: 200,
   CREATED: 201,
   UNAUTHORIZED: 401,
+  INCORRECT_PASSWORD: 401,
```

Comment:
This enum is for standard http codes, not for mapping http codes to our functionality
```suggestion
```

Reply:
Fixed it!